packages/error-reporting/package.json

```
@@ -0,0 +1,55 @@
+{
```

Why would we need to ignore the errors directory for unit-tests -- couldn't we run the errors tests as well?